modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -164,6 +168,18 @@ def update_status!(status:, code: nil, detail: nil)
       email_handler.handle! if status == 'submitted' && email_identifier.present?
     end
 
+    def extension_request?
+      form_data&.dig('data', 'attributes', 'extensionRequest')
```

Comment:
We already have `data_attributes` to get to the form_data's attributes :)

Reply:
😑